.env.example

```
@@ -1,57 +1,77 @@
+# Application client name
+KEYCLOAK_CLIENT_NAME_APP=your_client_name
+# Keycloak admin username
+KEYCLOAK_ADMIN_USERNAME=admin
+# Keycloak client ID
+KEYCLOAK_CLIENT_ID=your_client_id
+# OAuth grant type
+KEYCLOAK_GRANT_TYPE=client_credentials
+
+# Hasura Configuration
+HASURA_URL=http://localhost:8080/v1/graphql
+HASURA_GRAPHQL_ADMIN_SECRET=your_hasura_admin_secret
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The HASURA_GRAPHQL_ADMIN_SECRET key should go before the HASURA_URL key

(UnorderedKey)